packages/functional-tests/tests/settings/passwordVisibility.spec.ts

```diff
@@ -1,3 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 import { test, expect } from '../../lib/fixtures/standard';
 
 test.describe('password visibility tests', () => {
```

Comment:
Ah, I think this is what I was trying to get at before. This test/suite isn't flagged as `severity-1` or `severity-2` so when would it get executed?

Reply:
This will only run for local. Since its low priority, its not marked sev 1 or sev 2.